wormhole-connect/src/components/DemoApp/index.tsx

```diff
@@ -0,0 +1,127 @@
+import './styles.css';
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import WormholeConnect from '../../WormholeConnect';
+import { WormholeConnectConfig } from 'config/types';
+import { compressToBase64, decompressFromBase64 } from 'lz-string';
+
+const MAX_URL_SIZE = 30_000; // 30kb (HTTP header limit is set to 32kb)
+
+const parseConfig = (config: string): WormholeConnectConfig => {
+  if (config) {
+    try {
+      return eval(
+        `(function() { return ${config} })()`,
```

Comment:
Are we calling `eval` here to allow javascript in the config (e.g. a `WormholeConnectEventHandler` function?)

Reply:
that, and also just to allow us to copy & paste WIP source code in here, which is usually not valid JSON with string keys. eg `{ nttGroups: {...} }` vs `{ "nttGroups": {...} }`

Reply:
JSON also screams about trailing commas, etc. it's just annoyingly strict.